sympy/functions/special/spherical_harmonics.py

```diff
@@ -36,8 +37,8 @@ class Ynm(Function):
 
     >>> from sympy import Ynm, Symbol, simplify
     >>> from sympy.abc import n,m
-    >>> theta = Symbol("theta")
-    >>> phi = Symbol("phi")
+    >>> theta = Symbol("theta", real=True)
```

Comment:
Is this needed? It doesn't look like any of the added code makes use of it.

Reply:
They're to ensure `theta` and `phi` are real but the check in `_eval_conjugate` isn't working properly.